wordpress-mobile/WordPress-iOS
# Update Gemfile and Rakefile for newer dependencies.

Fixes #6124 

To test:
1. Edit the Rakefile to change the Swiftlint version to `master`.
2. Run `rake dependencies` to update everything. Verify CocoaPods 1.1.1 is installed.
3. Revert the changes to Rakefile.
4. Clean and build the workspace.

Needs review: @koke
